cmd/amppkg_dl_sxg/main.go

```diff
@@ -23,7 +23,7 @@ func getSXG(url string) ([]byte, error) {
 	if err != nil {
 		return nil, errors.WithStack(err)
 	}
-	req.Header.Set("Accept", "application/signed-exchange;v=b2")
+	req.Header.Set("Accept", "application/signed-exchange;v=b3")
 	req.Header.Set("AMP-Cache-Transform", "any")
```

Comment:
(Just as an FYI; Google's AMP Cache will begin sending `google;v="1"` instead of "any", with the "1" eventually becoming "1..2" and then "M..N" for incrementing M and N over time. But I don't think it's valuable to burden the version release process with the additional step of updating this debug binary, so I think leaving this as "any" is the best course of action.)

Reply:
Thanks for your information. I found that it follows the spec of https://github.com/ampproject/amphtml/blob/master/spec/amp-cache-transform.md#request-header.
If multiple versions of Transformer are supported at the same time in the future, I think it would be good to have a new command line option to overwrite the `AMP-Cache-Transform` header.